File: 2864. Maximum Odd Binary Number/maximumOddBinaryNumber.py
class Solution:
  def maximumOddBinaryNumber(self, s: str) -> str:
    count = 0

    for char in s:
      if char == '1':
        count += 1

    res = (count - 1) * '1' + (len(s) - count) * '0' + '1'
    return res
  
solution = Solution()
s = "0101"
print('maximumOddBinaryNumber: ', solution.maximumOddBinaryNumber(s))

File: 2864. Maximum Odd Binary Number/maximumOddBinaryNumber.js
/**
 * @param {string} s
 * @return {string}
 */
const maximumOddBinaryNumber = (s) => {
  let count = 0;

  for (let i = 0; i < s.length; i++) {
    if (s[i] === '1') {
      count++;
    }
  }

  const res = '1'.repeat(count - 1) + '0'.repeat(s.length - count) + '1';

  return res;
};

const s = "0101"
console.log('maximumOddBinaryNumber: ', maximumOddBinaryNumber(s))
